### PR TITLE
Pinned prestor's pandoc dependency to 1.19.2 since 2.x removes an opt…

### PR DESCRIPTION
**Pull request by tfenne**
…ion that is used.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Just a small update to a recipe to pin the version of pandoc that is used.  Without the pin pandoc resolves to pandoc 2.x which removed an option that prestor uses.

### CODE REVIEW

File: recipes/r-prestor/meta.yaml [hunk 1]
```diff
@@ -47,7 +47,7 @@ requirements:
     - r-bibtex
     - r-captioner
     - r-alakazam
-    - pandoc
+    - pandoc 1.19.2
```
Comment (bgruening) [inline recipes/r-prestor/meta.yaml]: can we just pin to <2.0

Comment (tfenne) [reply recipes/r-prestor/meta.yaml]: Done!

**Comment by bgruening** [comment]
@tfenne do you have time to grep through the source and fix other recipes as well?

**Comment by bgruening** [comment]
Thanks!

**Pull request merged**